sui/operators.js

```diff
@@ -0,0 +1,141 @@
+    const [receiver, amount] = args;
+
+    await callContract(
+        keypair,
+        client,
+        config,
+        chain,
+        gasServiceConfig.address,
+        'gas_service::collect_gas',
+        [gasServiceConfig.objects.gas_service, receiver, amount],
+        {
+            ...options,
+            capIndex: 0,
+        },
+    );
+
+    printInfo('Gas collected successfully');
+}
+
+async function refundGas(keypair, client, config, chain, args, options) {
+    if (!chain.contracts.gas_service) {
+        throw new Error('Gas service package not found.');
+    }
+
+    const gasServiceConfig = chain.contracts.gas_service;
+    const [messageId, receiver, amount] = args;
+
+    await callContract(
+        keypair,
+        client,
+        config,
+        chain,
+        gasServiceConfig.address,
+        'gas_service::refund',
+        [gasServiceConfig.objects.gas_service, bcs.string().serialize(messageId).toBytes(), receiver, amount],
+        {
+            ...options,
+            capIndex: 0,
+        },
+    );
+
+    printInfo('Gas refunded successfully');
+}
+
+async function mainProcessor(processor, args, options) {
+    const config = loadSuiConfig(options.env);
+
+    const [keypair, client] = getWallet(config.sui, options);
+    await printWalletInfo(keypair, client, config.sui, options);
+
+    await processor(keypair, client, config, config.sui, args, options);
+}
+
+if (require.main === module) {
+    const program = new Command();
+
+    program.name('operators').description('Operators contract operations.');
+
+    const callContractCmd = program
+        .command('call-contract <contractId> <functionName> [functionArgs...]')
+        .description('Call a contract with an optional borrowed capability')
+        .addOption(new Option('--capId <capId>', 'ID of the capability to borrow'))
+        .addOption(new Option('--capIndex <capIndex>', 'Index of the borrowed capability in the function arguments'))
+        .action((contractId, functionName, functionArgs, options) => {
+            options.capIndex = options.capIndex ? parseInt(options.capIndex, 10) : undefined;
+            mainProcessor(callContract, [contractId, functionName, functionArgs], options);
+        });
+
+    const collectGasCmd = program
+        .command('collect-gas <receiver> <amount>')
+        .description('Collect gas from the gas service')
+        .addOption(new Option('--capId <capId>', 'ID of the GasCollectorCap to borrow'))
+        .action((receiver, amount, options) => mainProcessor(collectGas, [receiver, amount], options));
+
+    const refundGasCmd = program
+        .command('refund-gas <messageId> <receiver> <amount>')
+        .description('Refund gas from the gas service')
+        .addOption(new Option('--capId <capId>', 'ID of the GasCollectorCap to borrow'))
+        .action((messageId, receiver, amount, options) => mainProcessor(refundGas, [messageId, receiver, amount], options));
+
+    addBaseOptions(program);
```

Comment:
This will give error that -p option is not specified when providing options via CLI instead of an env. As you are calling `addBaseOptions` with the program, as well as with other commands, it is trying to fetch `privateKey` options twice from CLI when running a command, this will not be a problem when base options are provided via an env file, as it can fetch the same options any number of times from env. So I believe `addBaseOptions` call to the program can be removed if it does not create a problem, so the base options can be provided via CLI as well.

Reply:
@npty is this still an issue?

Reply:
Yes, this is still an issue. will investigate.

Reply:
Fixed https://github.com/axelarnetwork/axelar-contract-deployments/pull/272/commits/27f1db3542e75f4344fda6e80c975de13987e0b4